assets/airports/ltba.json

```diff
@@ -1,254 +1,6868 @@
+        "MAKOL": ["N42d10.23", "E029d08.57"],
+        "MARMA": ["N40d31.63", "E027d38.93"],
+        "MIMTO": ["N41d50.82", "E33d2.80"],
+        "MISRO": ["N39d57m10", "E029d48m04"],
+        "MOPIN": ["N41d27.89", "E029d42.63"],
+        "NEVAL": ["N40d48.94", "E25d6.51"],
+        "NUMSO": ["N40d55m22", "E030d20m43"],
+        "OKBAL": ["N40d49.98", "E027d56.49"],
+        "OKESA": ["N37d37.75", "E27d23.18"],
+        "OLIDA": ["N38d25.10", "E24d53.47"],
+        "OSTEX": ["N42d9.57", "E26d57.36"],
+        "OTKOT": ["N42d6.68", "E34d25.97"],
+        "OVACI": ["N39d47.33", "E29d9.25"],
+        "PAZAR": ["N41d17.93", "E029d59.97"],
+        "PEMET": ["N41d18.71", "E029d04.27"],
+        "PEREN": ["N40d35.80", "E23d58.07"],
+        "PETAR": ["N40d10.27", "E32d4.88"],
+        "PIMAV": ["N40d24.04", "E029d57.22"],
+        "PINDO": ["N40d28.85", "E20d57.35"],
+        "REVDA": ["N43d44.00", "E29d8.60"],
+        "RILEX": ["N41d58.27", "E26d45.98"],
+        "RIMBO": ["N41d22.86", "E028d33.14"],
+        "RIXEN": ["N42d4.93", "E28d45.97"],
+        "ROBLO": ["N41d01.65", "E028d40.65"],
+        "RORKI": ["N43d5.77", "E29d8.60"],
+        "SADIK": ["N40d40.72", "E028d49.07"],
+        "SALGO": ["N40d28m26", "E032d11m58"],
+        "SBH"  : ["N40d53.98", "E29d19.22"],
+        "SERCE": ["N41d34.63", "E27d18.54"],
+        "SIN"  : ["N42d1.34", "E35d4.61"],
+        "SIVLI": ["N40d59.05", "E027d48.28"],
+        "SOMEP": ["N41d10.61", "E028d40.11"],
+        "SOMGU": ["N39d33.20", "E29d57.33"],
+        "SOSUS": ["N40d44.65", "E25d4.40"],
+        "SUDES": ["N39d46m10", "E026d53m36"],
+        "SULTA": ["N40d02m44", "E029d44m23"],
+        "SUTIS": ["N40d42.11", "E24d44.91"],
+        "TEMGO": ["N41d09.98", "E029d20.79"],
+        "TETSA": ["N41d44.77", "E028d46.75"],
+        "TEVNI": ["N41d35.48", "E30d21.83"],
+        "TIMUR": ["N43d41.33", "E24d14.35"],
+        "TIRER": ["N41d8.18", "E28d23.05"],
+        "TOKER": ["N40d18.80", "E30d46.33"],
+        "TOPLU": ["N41d28.72", "E028d10.39"],
+        "TUDBU": ["N41d58.00", "E027d46.62"],
+        "TURKO": ["N40d38.97", "E027d35.20"],
+        "UBEXA": ["N41d17.24", "E029d12.12"],
+        "ULMAR": ["N41d12.67", "E027d11.22"],
+        "ULMIP": ["N41d39.14", "E30d41.22"],
+        "ULPER": ["N40d41.75", "E029d39.15"],
+        "UMRUN": ["N40d11.17", "E31d52.68"],
+        "UNSAV": ["N41d28.70", "E028d47.38"],
+        "UNTUT": ["N41d35.55", "E028d41.01"],
+        "UPAMA": ["N43d37.58", "E25d25.10"],
+        "UVUDA": ["N42d13.52", "E27d48.22"],
+        "VADEN": ["N42d03.93", "E027d12.97"],
+        "VATOM": ["N40d59.14", "E029d06.44"],
+        "VIKBI": ["N44d15.51", "E24d39.35"],
+        "VURAL": ["N41d05.68", "E028d48.28"],
+        "YAA"  : ["N40d28.50", "E029d12.45"],
+        "YASEN": ["N41d11.93", "E029d30.97"],
+        "YAVRU": ["N40d34m56", "E031d49m58"],
+        "ZEMBA": ["N40d57.02", "E027d51.43"]
     },
-    "arrivals": [
+    "restricted": [
```

Comment:
I didn't add those because these areas are only restricted for helicopter flights and/or vfr traffics and we only have IFR flights and no helicopters. Should they be added eventhough?

Reply:
Nope that's fine :+1: